projects/core/src/asm/http-interceptors/csagent-auth-error.interceptor.ts

```diff
@@ -39,7 +41,10 @@ export class CustomerSupportAgentAuthErrorInterceptor
       catchError((errResponse: any) => {
         if (errResponse instanceof HttpErrorResponse) {
           // Unauthorized
-          if (isCustomerSupportAgentRequest && errResponse.status === 401) {
+          if (
+            isCustomerSupportAgentRequest &&
+            errResponse.status === ERROR_STATUS_UNAUTHORIZED
```

Comment:
we have statuses already defined in `core\src\global-message\models\response-status.model.ts`
```suggestion
            errResponse.status === HttpResponseStatus.UNAUTHORIZED
```